### ISSUE TITLE
For OSX the zap.sh in the core download hard-codes the relative java location

### ISSUE THREAD

**Issue by pwolanin** [zaproxy/zaproxy#1771]
see:

https://github.com/zaproxy/zaproxy/blob/develop/src/zap.sh#L86

The path 
```
  exec ../PlugIns/jre*/Contents/Home/bin/java 
```

Is (at best) only going to work if you you got the version with java included

At least locally, it works if I change to:

```
  exec {$JAVA_HOME}/bin/java 
```

This depends on having this in my .profile, or you could possibly add it to the script:
```
export JAVA_HOME=$(/usr/libexec/java_home)
```

or it would probably be fine to just use the java in the path.

At the least, there needs to be better doc with the core download for use cross-platform.  The README says double clicking the script will work, which doesn't seem to be true.

**Comment by marumari** [comment]
Sorry about that, @pwolanin!  #1773 should fix the issue you're having.  Perhaps @psiinon could recreate the 2.4.1 OS X build and re-upload it?

**Comment by marumari** [comment]
Fixed in #1773, should work with the built-in Java, and that's not there (as in zap.sh), will then use the system Java, provided that it's new enough.
